flake.nix

```diff
@@ -196,10 +201,11 @@
     };
 
     hydraJobs = with nixpkgs.lib; let
-      hydraJobs = mapAttrs (_: hydraJob);
+      #hydraJobs = mapAttrs (_: hydraJob);
     in {
-      checks = hydraJobs self.checks.x86_64-linux;
-      packages = hydraJobs self.packages.x86_64-linux;
+      # TODO: this does not evaluate (meta attribute is missing)
```

Comment:
Yet they worked: https://hydra.hq.c3d2.de/jobset/c3d2/beherbergung#tabs-jobs

Reply:
`nix flake check -vL` is broken if I add those lines back.